### PR TITLE
Allow nans to print in solution

### PR DESCRIPTION
**Pull request by pgkirsch**
To allow printing of `nans` in solution.

### CODE REVIEW

**Comment by pgkirsch** [comment]
Related to issue 269

File: gpkit/small_scripts.py [hunk 1]
```diff
@@ -189,7 +189,8 @@ def results_table(data, title, minval=0, printunits=True, fixedcols=True,
     decorated = [(bool(v.shape) if isinstance(v, Iterable) else False,
                  (varfmt % k),
                  i, k, v) for i, (k, v) in enumerate(data.items())
-                 if np.max(abs(v)) >= minval]
+                 if (np.max(abs(v)) >= minval) | (np.any(np.isnan(v)))]
+                #                               allows nans to be printed
```
Comment (whoburg) [inline gpkit/small_scripts.py]: Let's change the pipe (`|`) to `or` -- I think it's more readable.

Comment (whoburg) [inline gpkit/small_scripts.py]: @pgkirsch, agreed this list comprehension is dense and ugly. Refactoring table printing is definitely on the todo list! Just lower priority.

**Comment by whoburg** [comment]
+1 for merging as a hotfix for #269.

Longer term, @bqpd, should we discuss whether `nan`s should *ever* be in a `GPSolutionArray`? I'd be in favor of raising exceptions long before table printing if there are any nans, and therefore removing this fix. If you agree, let's still merge this but open a new issue for eliminating nans for a later milestone.

**Comment by whoburg** [comment]
PS note inline commit comment to address before merging.

**Comment by pgkirsch** [comment]
Made the or change as request.

And yeah, you're right there really shouldn't be `nans` there in the first place- I guess I just carried away with actually doing something.

**Comment by whoburg** [comment]
Actually doing something is great!! Continuous improvement...

**Comment by whoburg** [comment]
might want to close this and make the `or` change on #300 ?

**Comment by bqpd** [comment]
@whoburg good point. I haven't reached that point in the refactor. We could potentially use a sparse vector instead? I put `nan` there here just so it'd be obvious if something weird happened (even several steps of algebra later), which seems to have worked...

**Pull request closed without merging**